parlai/core/params.py

```diff
@@ -54,6 +54,17 @@ def modelzoo_path(datapath, path):
     if not path.startswith('models:'):
         return path
     else:
+        # Check if we need to download the model
+        animal = path[7:path.rfind('/')].replace('/', '.')
+        module_name = "parlai.zoo.%s" % (animal)  
```

Comment:
also this is cool, I saw this in pierre-emmanuel's diff, you can do this instead:
```
>>> w = 'world'
>>> f'hello {w}'
'hello world'
```

(e.g. `f'parlai.zoo.{animal}'` which I think is super clean and nice)